README.md

```diff
@@ -52,13 +56,13 @@ for the user-handled portions of SAML profiles. See "Metadata" and "Plugins" for
 * If testing an IdP:
   * Provide your IdP's metadata file path to the `samlconf` script using `-i` or `--idpMetadata`.
   * Configure your IdP with the test kit's SP metadata from
-  `<PATH_TO_PROJECT>/distribution/command-line/src/main/resources/test-sp-metadata.xml`
+  `distribution/command-line/src/main/resources/test-sp-metadata.xml`
   or `samlconf-1.0-SNAPSHOT/conf/test-sp-metadata.xml` from the distribution.
   
 * **TODO - not implemented yet** If testing an SP:
   * Provide your SP's metadata file path to the `samlconf` script using `-s` or `--spMetadata`.
   * Configure your SP with the test kit's IdP metadata from
-    `<PATH_TO_PROJECT>/distribution/command-line/src/main/resources/test-idp-metadata.xml`
+    `distribution/command-line/src/main/resources/test-idp-metadata.xml`
```

Comment:
✏️ Should really reference the `distribution/command-line/build/install/samlconf/conf` location (assuming `buildDist` was run, or just the `samlconf/conf` directory if an archive was exploded.